habitat/core/dataset.py

```diff
@@ -65,6 +67,25 @@ class Dataset(Generic[T]):
     """
     episodes: List[T]
 
+    @staticmethod
+    def _scene_from_episode(episode: T) -> str:
+        r"""Helper method to get the scene name from an episode.  Assumes
+        the scene_id is formated /path/to/<scene_name>.<ext>
+        """
+        return os.path.splitext(os.path.basename(episode.scene_id))[0]
+
+    @classmethod
+    def get_scenes_to_load(cls, config: Config) -> List[str]:
```

Comment:
Duplicating `scene_ids ` property: map(dataset.scene_ids, Dataset. _scene_from_episode). We should have one method for that maybe with param or combine 2 methods like above.

Reply:
The `.scene_ids` property is a little different, it returns the full scene paths, not just the scene name.  We could change that property, but I wasn't sure what the implications of doing that was.

Reply:
More detailed doc string required for the core class.